.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+**The title of this pull-request should be a brief description of what the pull-request fixes/improves/changes. Ideally 50 characters or less.**
```

Comment:
If this is going to show up like so in a PR, then I, personally, would want the title to stand out more - like a header does (<h1> or #).  That being said, I don't feel even remotely strongly about that.  Nice work!